src/Select/Select.component.tsx

```diff
@@ -184,6 +185,13 @@ export interface Props extends React.HTMLAttributes<HTMLDivElement> {
    * @default false
    **/
   highlightFilled?: boolean;
+
+  /**
+   * Placeholder text
```

Comment:
Please describe the new option. It is not clear now.

Reply:
fixed!